docs/build/quick_start.md

```diff
@@ -0,0 +1,223 @@
+# Quick start
+
+This guide provides short information for developers to start to work with sc-machine quickly. You can always learn more about the sc-machine's [build system](build_system.md).
+
+## Check CMake
+
+Ensure you are using **CMake version 3.24** or newer. Verify your version with:
+
+```sh
+cmake --version
+```
+
+To upgrade CMake, follow the installation guide appropriate for your OS or use:
+  
+```sh
+# Use pipx to install cmake if not already installed
+# Install pipx first using guide: https://pipx.pypa.io/stable/installation/
+pipx install cmake
+```
+
+## Start develop sc-machine with Conan
+
+### Install Conan
+
+Install Conan, to build sc-machine dependencies with Conan-provided dependencies:
+
+```sh
+# Use pipx to install conan if not already installed
+pipx install conan
+```
+
+### Use sc-machine in Debug
+
+#### Build sc-machine in Debug
+
+To build sc-machine in debug mode using Conan-provided dependencies, run:
+
+```sh
+# debug build type
+cmake --preset debug-conan
+cmake --build --preset debug
+```
+
+Note: By default, configure preset `debug` enables building sc-machine tests.
+
+#### Run sc-machine tests in Debug
+
+After that, you can go to `build/Debug` and run tests via `ctest`:
+
+```sh
+cd build/Debug
+ctest -C Debug -V
```

Comment:
`Debug` indicates that the tests should be executed using the Debug configuration, which usually includes debugging symbols and disables optimizations to facilitate easier debugging.

Reply:
The Release configuration typically includes compiler optimizations, which means that the code will run faster compared to the Debug configuration, where optimizations are usually disabled to facilitate debugging.

Reply:
`man ctest` is saying that -C has some effect in case there are multiple builds, but documentation is asking to change-directory to build/Debug so there is only Debug configuration to choose from
`
Some  CMake-generated build trees can have multiple build configurations in the same tree.  This option can be used to specify which one should be tested.
`

Reply:
Okay